index.html

```diff
@@ -38,12 +37,191 @@ <h1 class="greetings">
           <h3>LET’S CONNECT</h3>
         </a>
         <br />
+        <ul class="social-icons margin-top-12">
+          <li>
+            <a href="#">
+            <img
+              class="social twitter"
+              src="image/icons/twitter.png"
+              alt="Twitter icon"
+              />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+            <img
+              class="social linkedin"
+              src="image/icons/linkedin.png"
+              alt="linkedin icon"
+              />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+            <img class="social m" src="image/icons/m.png" alt="m icon" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+            <img
+              class="social github"
+              src="image/icons/github.png"
+              alt="github icon"
+              />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+            <img
+              class="social fin"
+              src="image/icons/fin.png"
+              alt="fin icon"
+              />
+            </a>
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <section class="works">
+      <div class="grid works-grid-container wrap padding-top-0">
+        <div class="flex works-grid-item">
+          <img
+            class="width-100"
+            src="image/tonic.jpg"
+            alt="portfolio screenshot"
+          />
+          <h3 class="section-title margin-top-12">portfolio</h3>
+          <div class="work-period">
+            <ul>
+              <li class="text-color-primary">Canopy</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p class="work-description text-color-primary margin-top-20">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="work-categories margin-top-12 padding-0">
+            <li><span>html</span></li>
+            <li><span>css</span></li>
+            <li><span>javascript</span></li>
+          </ul>
+          <div>
+          <button type="button" class="button button-enabled">
```

Comment:
- [ ] please align this item so they look like the original [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) version. follow the same line spacing and font characteristics.
your work 👇🏾
![Screenshot (58)_LI](https://user-images.githubusercontent.com/65298048/170492864-b2b414b5-d4ac-40a1-8361-aeaa1077445f.jpg)
figma design 👇🏾 

![Screenshot (57)_LI](https://user-images.githubusercontent.com/65298048/170492965-0cc5f4f5-b3b8-4220-bf44-1c38f3f79e01.jpg)